guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,40 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== Granular Smart Proxy Content Counts
+
+You can now update smart proxy content counts per lifecycle environment and per content view.
+Smart proxy auto-syncs will now update only the lifecycle environment rather than all counts.
+A substantial performance improvement is expected.
+
+=== Multiple content view environments for hosts and activation keys
+
+Hosts and (new in 4.15) activation keys can now be associated with multiple content view environments. To use this feature, turn on the `allow_multiple_content_views` Setting.
```

Comment:
There are still cases in this paragraph where it's not using one sentence per line. I won't highlight them all, but it follows from https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#pull-request-checklist:
> I put each sentence to its own line.

```suggestion
Hosts and (new in 4.15) activation keys can now be associated with multiple content view environments.
To use this feature, turn on the `allow_multiple_content_views` Setting.
```